<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="MNCTVN05589" name="Default test" tests="5" failures="0" timestamp="17 Jun 2020 06:00:17 GMT" time="0.184" errors="0">
  <testcase name="@BeforeClass setUpClass" time="0.0" classname="test.Test_Chat">
    <failure type="cucumber.runtime.CucumberException" message="No backends were found. Please make sure you have a backend module on your CLASSPATH.">
      <![CDATA[cucumber.runtime.CucumberException: No backends were found. Please make sure you have a backend module on your CLASSPATH.
	at cucumber.runtime.Runtime.<init>(Runtime.java:81)
	at cucumber.runtime.Runtime.<init>(Runtime.java:70)
	at cucumber.runtime.Runtime.<init>(Runtime.java:66)
	at cucumber.api.testng.TestNGCucumberRunner.<init>(TestNGCucumberRunner.java:42)
	at config.Setup.setUpClass(Setup.java:29)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:514)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:215)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:178)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
    </failure>
  </testcase> <!-- @BeforeClass setUpClass -->
  <testcase name="@BeforeMethod set" time="0.0" classname="test.Test_Chat">
    <skipped/>
  </testcase> <!-- @BeforeMethod set -->
  <testcase name="@AfterMethod done" time="0.0" classname="test.Test_Chat">
    <skipped/>
  </testcase> <!-- @AfterMethod done -->
  <testcase name="@BeforeMethod set" time="0.0" classname="test.Test_Chat">
    <skipped/>
  </testcase> <!-- @BeforeMethod set -->
  <testcase name="@AfterMethod done" time="0.0" classname="test.Test_Chat">
    <skipped/>
  </testcase> <!-- @AfterMethod done -->
  <testcase name="@BeforeMethod set" time="0.0" classname="test.Test_Chat">
    <skipped/>
  </testcase> <!-- @BeforeMethod set -->
  <testcase name="@AfterMethod done" time="0.0" classname="test.Test_Chat">
    <skipped/>
  </testcase> <!-- @AfterMethod done -->
  <testcase name="@BeforeMethod set" time="0.0" classname="test.Test_Chat">
    <skipped/>
  </testcase> <!-- @BeforeMethod set -->
  <testcase name="@AfterMethod done" time="0.0" classname="test.Test_Chat">
    <skipped/>
  </testcase> <!-- @AfterMethod done -->
  <testcase name="@AfterClass setDownClass" time="0.0" classname="test.Test_Chat">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at config.Setup.setDownClass(Setup.java:44)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:514)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:215)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.internal.TestMethodWorker.invokeAfterClassMethods(TestMethodWorker.java:232)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:115)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
    </failure>
  </testcase> <!-- @AfterClass setDownClass -->
  <testcase name="" time="0.0" classname="test.Test_Chat">
    <skipped/>
  </testcase> <!--  -->
  <testcase name="" time="0.002" classname="test.Test_Chat">
    <skipped/>
  </testcase> <!--  -->
  <testcase name="" time="0.002" classname="test.Test_Chat">
    <skipped/>
  </testcase> <!--  -->
  <testcase name="" time="0.001" classname="test.Test_Chat">
    <skipped/>
  </testcase> <!--  -->
  <testcase name="" time="0.002" classname="test.Test_Chat">
    <skipped/>
  </testcase> <!--  -->
</testsuite> <!-- Default test -->
